templateChecks.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+TEMPLATE_BIG_BANG_VERSION=$(sed -n -e "s/\.*final BIGBANG_VERSION = '\s*\(.*\)[\"\']\s*/\1/p" template/app/build.gradle)
+BIG_BANG_VERSION=$(sed -n -e "s/\.*librariesVersion = '\s*\(.*\)[\"\']\s*/\1/p" versions.gradle)
+
+if [ ${TEMPLATE_BIG_BANG_VERSION} = ${BIG_BANG_VERSION} ]; then
```

Comment:
I think one is ok, if you want you can use `-eq` too, but I think that `=` is ok